README.md

```diff
@@ -2,9 +2,16 @@
 
 In this repository, you can find the lang files for the framework PHP, [Laravel 4&5](http://www.laravel.com).
 
-## How add languages in my app ?
+## Install
 
-### Installation by GitHub
+#### Via Composer
+ * For Laravel 4 : add `"caouecs/laravel4-lang": "~1.0"` in your `composer.json` in "require" or run *composer require caouecs/laravel4-lang*
```

Comment:
you forget *For Laravel 5.2 : add `"caouecs/laravel-lang": "~3.0"` in your `composer.json` in "require"*